tests/safeds/data/tabular/transformation/test_sequential_table_transformer.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.transformation import (
+    Discretizer,
+    LabelEncoder,
+    OneHotEncoder,
+    SequentialTableTransformer,
+    SimpleImputer,
+    StandardScaler,
+    TableTransformer,
+)
+from safeds.exceptions import TransformerNotFittedError, TransformerNotInvertibleError
+
+from tests.helpers import assert_tables_equal
+
+
+class TestInit:
+
+    def test_should_raise_value_error_on_empty_list(self) -> None:
+        with pytest.raises(ValueError, match=("transformers must contain at least 1 transformer")):
+            SequentialTableTransformer(transformers=[])  # type: ignore  # noqa: PGH003
+
+
+class TestFit:
+    def test_should_raise_value_error_on_empty_table(self) -> None:
+        one_hot = OneHotEncoder()
+        imputer = SimpleImputer(SimpleImputer.Strategy.constant(0))
+        transformers = [one_hot, imputer]
+        test_table = Table(
+            {
+                "col1": [],
+                "col2": [],
+            },
+        )
+        sequential_table_transformer = SequentialTableTransformer(transformers)
+        with pytest.raises(
+            ValueError,
+            match=("The SequentialTableTransformer cannot be fitted because the table contains 0 rows."),
+        ):
+            sequential_table_transformer.fit(test_table)
+
+    def test_fit_does_not_change_original_transformer(self) -> None:
```

Comment:
```suggestion
    def test_should_not_change_original_transformer(self) -> None:
```

The method under test is already specified by the wrapping class.

Reply:
changed name of test to the one Suggested